src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -52,7 +52,7 @@ def load_arguments(self, _):
                                      help="Scale set name. You can configure the default using `az configure --defaults vmss=<name>`",
                                      id_part='name')
 
-    extension_instance_name_type = CLIArgumentType(help="Name of the vm's instance of the extension. Default: name of the extension.")
+    extension_instance_name_type = CLIArgumentType(help="Name of extension instance, which can be customized. Default: name of the extension.")
```

Comment:
It looks an alias of an extension?

Reply:
No. It's the name that can be customized by user. `--name` is type of extension.
Nice writing.

Reply:
But you don't change the help message? Do you have any other concern?

Reply:
The two sentences have same meanings.